src/components/Section.jsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react"
+import styled from "styled-components"
+
+import { Grid } from "../components/Grid"
+
+const Container = styled.div`
+  background-color: var(--color-heading);
+  padding: var(--spacing-24);
```

Comment:
Since `Grid` already has horizontal padding, you probably shouldn't give its parent (`Container`) any more horizontal padding. 

I think you need to give _vertical_ padding to `Container` instead:

```diff
-padding: var(--spacing-24);
+padding: var(--spacing-24) 0;
```

Reply:
I am not sure if it is right. If I will do it then in the smaller screens we are not going to have padding vertical.
Or I need to add your idea and fix this problem (with horizontal padding on the smaller screens) in the Grid instead. What do you think?
<img width="838" alt="Screenshot 2021-08-03 at 13 19 39" src="https://user-images.githubusercontent.com/33199975/128006926-1eda15d8-4662-471e-a632-86f3facecbe5.png">
<img width="837" alt="Screenshot 2021-08-03 at 13 20 33" src="https://user-images.githubusercontent.com/33199975/128007046-31ddc389-238d-4b06-bf25-5a062286f49f.png">

Reply:
Yeah, you should delegate horizontal padding to Grid because you always want that (so that the grid doesn't touch the edges of the screen). Then you can control vertical padding in components like this one (Container) _in addition_.